index.html

```diff
@@ -82,6 +82,7 @@
 // production code goes here</textarea>
     <p id="status"></p>
     <script type="text/javascript">
+(function iife() {
```

Comment:
@wenzowski I've found it useful to have the CodeMirror vars (`*Editor`) in the global scope. That way it's easy to experiment with the CodeMirror API from the console.

Maybe we could add a namespace to `window` to hold global references?